encryptonize/defs.go

```diff
@@ -34,56 +40,80 @@ type HealthResponse struct {
 type Scope int
 
 const (
-	ScopeRead Scope = iota
-	ScopeCreate
-	ScopeUpdate
-	ScopeDelete
-	ScopeIndex
-	ScopeObjectPermissions
-	ScopeUserManagement
+	ScopeRead              Scope = iota // Scope to decrypt objects
+	ScopeCreate                         // Scope encrypt/store objects
+	ScopeUpdate                         // Scope to update existing objects
+	ScopeDelete                         // Scope to delete stored objects
+	ScopeIndex                          // Scope to list objects
```

Comment:
Isn't ScopeIndex used to GetPermissions? https://github.com/cyber-crypt-com/encryptonize-core/blob/b2e824b0d9b540a797275031f52d01565628995d/common/middleware/authz.go#L46

Reply:
Right, fixed!